app/src/main/java/com/cg/lrceditor/LyricListAdapter.java

```diff
@@ -182,22 +182,34 @@ class LyricListItem extends RecyclerView.ViewHolder implements View.OnLongClickL
 				itemAdd.setImageDrawable(ctx.getDrawable(R.drawable.ic_add_light));
 			}
 
-			itemAdd.setOnClickListener(v -> clickListener.onAddButtonClick(getAdapterPosition()));
+			itemAdd.setOnClickListener(v -> {
+				clickListener.onAddButtonClick(getAdapterPosition());
+				v.performHapticFeedback(HapticFeedbackConstants.KEYBOARD_TAP);
+			});
 
-			itemPlay.setOnClickListener(v -> clickListener.onPlayButtonClick(getAdapterPosition()));
+			itemPlay.setOnClickListener(v -> {
+				clickListener.onPlayButtonClick(getAdapterPosition());
+				v.performHapticFeedback(HapticFeedbackConstants.KEYBOARD_TAP);
+			});
 
 			ImageButton incrTime = itemView.findViewById(R.id.increase_time_button);
 			ImageButton decrTime = itemView.findViewById(R.id.decrease_time_button);
 
-			incrTime.setOnClickListener(v -> clickListener.onIncreaseTimeClick(getAdapterPosition()));
+			incrTime.setOnClickListener(v -> {
+				clickListener.onIncreaseTimeClick(getAdapterPosition());
+				v.performHapticFeedback(HapticFeedbackConstants.KEYBOARD_TAP);
+			});
 
 			incrTime.setOnLongClickListener(v -> {
 				clickListener.onLongPressIncrTime(getAdapterPosition());
 				v.performHapticFeedback(HapticFeedbackConstants.LONG_PRESS);
 				return false;
 			});
 
-			decrTime.setOnClickListener(v -> clickListener.onDecreaseTimeClick(getAdapterPosition()));
+			decrTime.setOnClickListener(v -> {
+				clickListener.onDecreaseTimeClick(getAdapterPosition());
+				v.performHapticFeedback(HapticFeedbackConstants.KEYBOARD_TAP);
+			});
```

Comment:
I feel that the haptics on every button click is a bit too much. Lets revert these changes.